hack/multikueue-e2e-test.sh

```diff
@@ -52,13 +52,19 @@ function startup {
             mkdir -p "$ARTIFACTS"
         fi
 
-        MANAGER_KIND_CONFIG="${SOURCE_DIR}/multikueue/manager-cluster.kind-${KIND_VERSION}.yaml"
-        if [ !  -f "$MANAGER_KIND_CONFIG" ]; then
-            MANAGER_KIND_CONFIG="${SOURCE_DIR}/multikueue/manager-cluster.kind.yaml"
+	cp "${SOURCE_DIR}/multikueue/manager-cluster.kind.yaml" "$ARTIFACTS"
+
+        #Enable the JobManagedBy feature gates for k8s 1.30+ versions 
+        ifs_bk="$IFS"
+        IFS=. read -r -a varr <<< "$KIND_VERSION"
+        IFS="$ifs_bk"
+        minor=$(( varr[1] ))        
+        if [ "$minor" -ge 30 ]; then
```

Comment:
```suggestion
        if [ "$(echo "$KIND_VERSION" | awk -F '.' '{print $2}')" -ge 30 ]; then
```

Reply:
Sure there are alternatives including `sed -E 's/([0-9]+).([0-9]+).([0-9]+)/\2/'`, But I prefer to use only bash if possible.

Reply:
Make sense.

Reply:
On that case I would propose:

```suggestion
        IFS=. read -r -a varr <<< "$KIND_VERSION"
        minor=$(( varr[1] ))        
        if [ "$minor" -ge 30 ]; then
```

We directly use `IFS` for the `read` command.